### PR TITLE
Web site pages

### PR DESCRIPTION
**Pull request by lubnaabd**

### CODE REVIEW

**Comment by NouraldinS** [comment]
@lubnaabd Assign me when it's review is approved.

**Review by HemaSAli — APPROVED** [review]

File: public/css/cohortWebsite.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  text-align: center;
+  font-family: lato;
+  color: #389dbf;
+  font-size: 28px;
+  line-height: 43px;
+  letter-spacing: 5%;
+  font-weight: bold;
+}
+
+.CohortList {
```
Comment (ali-7) [inline public/css/cohortWebsite.css]: fix it

File: public/css/homeWebsite.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+.project {
+  text-align: center;
+  display: grid;
+  width: 80%;
+  grid-template-columns:   repeat(auto-fit, minmax(200px, 1fr));
+  grid-gap: 20%;
+  margin: 0 auto;
+}
+
+.C_project {
```
Comment (ali-7) [inline public/css/homeWebsite.css]: fix it

File: public/css/homeWebsite.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  height: 160px;
+  justify-self: center;
+  border-radius: 50%;
+  background-color: #ffa41f;
+  text-align: center;
+  margin: 0 auto;
+  transition-duration: 0.4s;
+}
+
+.C_project:hover {
```
Comment (ali-7) [inline public/css/homeWebsite.css]: fix it

File: public/css/homeWebsite.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  -webkit-box-shadow: 0px 0px 46px 7px rgba(255, 164, 31, 1);
+  -moz-box-shadow: 0px 0px 46px 7px rgba(255, 164, 31, 1);
+  box-shadow: 0px 0px 46px 7px rgba(255, 164, 31, 1);
+}
+
+.image {
+  margin-top: 45px;
+}
+
+.Community {
```
Comment (ali-7) [inline public/css/homeWebsite.css]: fix it

File: src/views/cohortPageWebsite.hbs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    <div class="comp">
+      <img class="projectImg" src="/img/startup-photos.jpg">
+      <h4>Project Name</h4>
+    </div>
+    <div class="comp">
+      <img class="projectImg" src="/img/startup-photos.jpg">
+      <h4>Project Name</h4>
+    </div>
+  </div>
+
```
Comment (ali-7) [inline src/views/cohortPageWebsite.hbs]: remove space

File: src/views/cohortPageWebsite.hbs [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    <div class="comp">
+      <img class="projectImg" src="/img//startup-photos.jpg">
+      <h4>Project Name</h4>
+    </div>
+    <div class="comp">
+      <img class="projectImg" src="/img//startup-photos.jpg">
+      <h4>Project Name</h4>
+    </div>
+  </div>
+
```
Comment (ali-7) [inline src/views/cohortPageWebsite.hbs]: remove space

File: src/views/homeWebsite.hbs [hunk 7]
```diff
@@ -0,0 +1,51 @@
+<section class="main">
+  <section class="slider">
+    <div class=" w3-display-container">
+      <img class="mySlides" src="/img/besart-bootcamp.jpg">
+      <img class="mySlides" src="/img/besart-bootcamp2.jpg">
+      <img class="mySlides" src="/img/besart-bootcamp3.jpg">
+
```
Comment (ali-7) [inline src/views/homeWebsite.hbs]: remove space

File: src/views/layouts/webSite.hbs [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
```
Comment (ali-7) [inline src/views/layouts/webSite.hbs]: fix indentation

File: src/views/partials/webSite/webSiteHead.hbs [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (ali-7) [inline src/views/partials/webSite/webSiteHead.hbs]: fix indentation

**Review by ali-7 — CHANGES_REQUESTED** [review]

**Review by ali-7 — APPROVED** [review]

**Review by NouraldinS — CHANGES_REQUESTED** [review]
Divide pull request into 2-3 more branches! 
This pull cannot be made for one and only one feature, reference the issue of the branch.

**Pull request merged**